kwok/start-clusters.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+/usr/local/bin/kwokctl start cluster --name kwok &
+/usr/local/bin/kwokctl start cluster --name kwok-m1 &
+/usr/local/bin/kwokctl start cluster --name kwok-m2
```

Comment:
the way this is written, the script may exit before "kwok" and "kwok-m1" finih starting if kwok-m2 finishes starting faster

Reply:
Yes. This one is problematic. I'll see if I can make it work without the script.

Reply:
I removed the use of the script.